A couple of tests failed, both with missing schema key or data disrepency:
```
======================================================================
FAIL: test_run (test_all_fields.TestSquareAllFields) (stream='refunds')
Instantiate start date according to the desired data set and run the test
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/project/tests/test_all_fields.py", line 192, in all_fields_test
    self.assertRecordsEqual(stream, expected_record, actual_record)
  File "/root/project/tests/base.py", line 649, in assertRecordsEqual
    self.assertDictEqual(expected_record, sync_record)
AssertionError: {'upd[165 chars]R', 'location_id': 'VEN5D4KBE9GTT', 'reason': [187 chars]SZY'} != {'upd[165 chars]R', 'reason': 'Becuase you are worth it', 'pay[325 chars]Z'}]}
  {'amount_money': {'amount': 4, 'currency': 'USD'},
   'created_at': '2020-11-23T19:02:26.335000Z',
   'id': 'FiinNT4GAuTGA6pXD0NA9YA48BLZY_FDsGHK6VaexE3p9UcP7hVExAiFoJ9x7qT8TCt3cUWvR',
   'location_id': 'VEN5D4KBE9GTT',
   'order_id': 'BeRXRl3KBgvOok5ihAKy7e6mkuSZY',
   'payment_id': 'FiinNT4GAuTGA6pXD0NA9YA48BLZY',
+  'processing_fee': [{'amount_money': {'amount': 0, 'currency': 'USD'},
+                      'effective_at': '2020-11-23T21:02:22.000000Z',
+                      'type': 'INITIAL'}],
   'reason': 'Becuase you are worth it',
   'status': 'COMPLETED',
   'updated_at': '2020-11-23T19:02:28.281000Z'}

======================================================================
FAIL: test_run (test_all_fields.TestSquareAllFields) (stream='payments')
Instantiate start date according to the desired data set and run the test
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/project/tests/test_all_fields.py", line 168, in all_fields_test
    msg="\nFields missing from schema: {}\n".format(expected_keys.difference(schema_keys))
AssertionError: Items in the second set but not the first:
'risk_evaluation' : 
Fields missing from schema: {'risk_evaluation'}
```
And
```
======================================================================
FAIL: test_run (test_bookmarks.TestSquareIncrementalReplication) (stream='payments')
Instantiate start date according to the desired data set and run the test
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/project/tests/test_bookmarks.py", line 406, in bookmarks_test
    self.assertRecordsEqual(stream, created_record, sync_record)
  File "/root/project/tests/base.py", line 643, in assertRecordsEqual
    self.assertDictEqualWithOffKeys(expected_record, sync_record, {'updated_at'})
  File "/root/project/tests/base.py", line 652, in assertDictEqualWithOffKeys
    self.assertParentKeysEqual(expected_record, sync_record)
  File "/root/project/tests/base.py", line 631, in assertParentKeysEqual
    "[expected_record={}][sync_record={}]".format(expected_record, sync_record))
AssertionError: Items in the first set but not the second:
'risk_evaluation' : Expected keys in expected_record to equal keys in sync_record. [expected_record={'note': '#payment_20201123190759127261Z', 'delayed_until': '2020-11-30T19:07:59.307000Z', 'location_id': 'VEN5D4KBE9GTT', 'id': '1qWxTdbX1uTdjoz2ZGm2PjVi2iJZY', 'receipt_url': 'https://squareupsandbox.com/receipt/preview/1qWxTdbX1uTdjoz2ZGm2PjVi2iJZY', 'order_id': 'HR1r8vy8xiXnCtFnj7FhQVs4QzQZY', 'refunded_money': {'amount': 7, 'currency': 'USD'}, 'delay_duration': 'PT168H', 'amount_money': {'amount': 8157, 'currency': 'USD'}, 'total_money': {'amount': 8157, 'currency': 'USD'}, 'updated_at': '2020-11-23T19:08:07.186000Z', 'created_at': '2020-11-23T19:07:59.307000Z', 'source_type': 'CARD', 'risk_evaluation': {'created_at': '2020-11-23T19:07:59.415000Z', 'risk_level': 'NORMAL'}, 'status': 'COMPLETED', 'processing_fee': [{'amount_money': {'amount': 267, 'currency': 'USD'}, 'effective_at': '2020-11-23T21:08:00.000000Z', 'type': 'INITIAL'}], 'receipt_number': '1qWx', 'refund_ids': ['1qWxTdbX1uTdjoz2ZGm2PjVi2iJZY_IRW3IREOryEkpaLLBSPwYORSLjdozoA8BdjLSgq7BGb'], 'delay_action': 'CANCEL', 'card_details': {'card': {'card_type': 'CREDIT', 'exp_month': 11, 'prepaid_type': 'NOT_PREPAID', 'exp_year': 2022, 'bin': '540988', 'fingerprint': 'sq-1-RYhbqPYVst2RlujAp1cW4ehKQuSjXJ4C8p_dYu0JiAu8YcR08dwzoLwJkWHbqqQy3g', 'card_brand': 'MASTERCARD', 'last_4': '9029'}, 'statement_description': 'SQ *DEFAULT TEST ACCOUNT', 'status': 'CAPTURED', 'avs_status': 'AVS_ACCEPTED', 'cvv_status': 'CVV_ACCEPTED', 'entry_method': 'KEYED'}}][sync_record={'note': '#payment_20201123190759127261Z', 'delayed_until': '2020-11-30T19:07:59.307000Z', 'location_id': 'VEN5D4KBE9GTT', 'id': '1qWxTdbX1uTdjoz2ZGm2PjVi2iJZY', 'receipt_url': 'https://squareupsandbox.com/receipt/preview/1qWxTdbX1uTdjoz2ZGm2PjVi2iJZY', 'order_id': 'HR1r8vy8xiXnCtFnj7FhQVs4QzQZY', 'refunded_money': {'amount': 7, 'currency': 'USD'}, 'delay_duration': 'PT168H', 'amount_money': {'amount': 8157, 'currency': 'USD'}, 'total_money': {'amount': 8157, 'currency': 'USD'}, 'updated_at': '2020-11-23T19:08:07.186000Z', 'created_at': '2020-11-23T19:07:59.307000Z', 'source_type': 'CARD', 'status': 'COMPLETED', 'processing_fee': [{'amount_money': {'amount': 267, 'currency': 'USD'}, 'effective_at': '2020-11-23T21:08:00.000000Z', 'type': 'INITIAL'}], 'receipt_number': '1qWx', 'refund_ids': ['1qWxTdbX1uTdjoz2ZGm2PjVi2iJZY_IRW3IREOryEkpaLLBSPwYORSLjdozoA8BdjLSgq7BGb'], 'delay_action': 'CANCEL', 'card_details': {'card': {'card_type': 'CREDIT', 'fingerprint': 'sq-1-RYhbqPYVst2RlujAp1cW4ehKQuSjXJ4C8p_dYu0JiAu8YcR08dwzoLwJkWHbqqQy3g', 'prepaid_type': 'NOT_PREPAID', 'exp_year': 2022, 'bin': '540988', 'exp_month': 11, 'card_brand': 'MASTERCARD', 'last_4': '9029'}, 'statement_description': 'SQ *DEFAULT TEST ACCOUNT', 'status': 'CAPTURED', 'avs_status': 'AVS_ACCEPTED', 'cvv_status': 'CVV_ACCEPTED', 'entry_method': 'KEYED'}}]

======================================================================
FAIL: test_run (test_bookmarks.TestSquareIncrementalReplication) (stream='refunds')
Instantiate start date according to the desired data set and run the test
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/project/tests/test_bookmarks.py", line 406, in bookmarks_test
    self.assertRecordsEqual(stream, created_record, sync_record)
  File "/root/project/tests/base.py", line 649, in assertRecordsEqual
    self.assertDictEqual(expected_record, sync_record)
AssertionError: {'pay[39 chars]Y', 'created_at': '2020-11-23T19:08:04.166000Z[313 chars]00Z'} != {'pay[39 chars]Y', 'order_id': 'XbylqbA5jYzvKj5SkLdjOYm54GWZY[452 chars]00Z'}
  {'amount_money': {'amount': 7, 'currency': 'USD'},
   'created_at': '2020-11-23T19:08:04.166000Z',
   'id': '1qWxTdbX1uTdjoz2ZGm2PjVi2iJZY_IRW3IREOryEkpaLLBSPwYORSLjdozoA8BdjLSgq7BGb',
   'location_id': 'VEN5D4KBE9GTT',
   'order_id': 'XbylqbA5jYzvKj5SkLdjOYm54GWZY',
   'payment_id': '1qWxTdbX1uTdjoz2ZGm2PjVi2iJZY',
+  'processing_fee': [{'amount_money': {'amount': -1, 'currency': 'USD'},
+                      'effective_at': '2020-11-23T21:08:00.000000Z',
+                      'type': 'INITIAL'}],
   'reason': 'Becuase you are worth it',
   'status': 'COMPLETED',
   'updated_at': '2020-11-23T19:08:07.186000Z'}

----------------------------------------------------------------------
Ran 4 tests in 310.459s

FAILED (failures=2)

Exited with code exit status 1
CircleCI received exit code 1
```

--- a/tap_square/schemas/items.json
+++ b/tap_square/schemas/items.json
@@ -276,6 +276,18 @@
             "string"
           ]
         },
+        "legacy_tax_ids": {
+          "type": [
+            "null",
+            "array"
+          ],
+          "items": {
+            "type": [
+              "null",
+              "string"
+            ]
+          }
+        },
         "tax_ids": {
           "type": [
             "null",

--- a/tap_square/schemas/payments.json
+++ b/tap_square/schemas/payments.json
@@ -326,6 +326,27 @@
         "string"
       ]
     },
+    "risk_evaluation": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "properties": {
+        "created_at": {
+          "type": [
+            "null",
+            "string"
+          ],
+          "format": "date-time"
+        },
+        "risk_level": {
+          "type": [
+            "null",
+            "string"
+          ]
+        }
+      }
+    },
     "receipt_url": {
       "type": [
         "null",

--- a/tests/test_client.py
+++ b/tests/test_client.py
@@ -820,7 +820,7 @@ class TestClient():
             LOGGER.error("payment attempted to be refunded: %s", payment_obj)
             raise RuntimeError(refund.errors)
 
-        completed_refund = self.get_object_matching_conditions('refunds', refund.body.get('refund').get('id'), start_date=start_date, status='COMPLETED')
+        completed_refund = self.get_object_matching_conditions('refunds', refund.body.get('refund').get('id'), start_date=start_date, keys_exist={'processing_fee'}, status='COMPLETED')
         completed_payment = self.get_object_matching_conditions('payments', payment_response.get('id'), start_date=start_date, keys_exist={'processing_fee'}, status='COMPLETED', refunded_money=amount_money)[0]
         return (completed_refund, completed_payment)
 

